### PR TITLE
Add go-redis support

### PR DESCRIPTION
**Pull request by kerkerj**
Add `go-redis` library support, please take a look. :)

### CODE REVIEW

File: store/goredisstore/goredisstore_test.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	"github.com/throttled/throttled/store/storetest"
+)
+
+const (
+	redisTestDB     = 1
+	redisTestPrefix = "throttled-go-redis:"
+)
+
+// Demonstrates that how to initialize a RateLimiter with redis
+// using go-redis libaray.
```
Comment (brandur) [inline store/goredisstore/goredisstore_test.go]: Thanks for the example! Typo here: "library".

Comment (kerkerj) [reply store/goredisstore/goredisstore_test.go]: Oops 😅. Fixed! 😸

**Comment by brandur** [comment]
Thanks again! Pulling this in.

**Comment by brandur** [comment]
Released as `v2.2.0`.

**Pull request merged**